fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/logical/LogicalScanOperator.java

```diff
@@ -50,7 +50,12 @@ public LogicalScanOperator(
         this.table = Objects.requireNonNull(table, "table is null");
         this.colRefToColumnMetaMap = ImmutableMap.copyOf(colRefToColumnMetaMap);
         this.columnMetaToColRefMap = ImmutableMap.copyOf(columnMetaToColRefMap);
+        setPredicate(predicate);
+    }
 
+    @Override
+    public void setPredicate(ScalarOperator predicate) {
+        super.setPredicate(predicate);
```

Comment:
I think we must call `ColumnFilterConverter.convertColumnFilter` in specific rule, not always update it when set predicate or construncate a operator